svd-encoder/Cargo.toml

```diff
@@ -13,7 +13,8 @@ readme = "README.md"
 
 [dependencies]
 convert_case = "0.6.0"
-svd-rs = { version = "0.14.7", path = "../svd-rs" }
+svd-rs = { repository = "https://github.com/romancardenas/svd.git" } # TODO use crates.io
```

Comment:
Why this change is here?

`path` is highest priority and works well locally and with CI.